types/graphql/Maybe.d.ts

```diff
@@ -0,0 +1,3 @@
+type Maybe<T> = null | undefined | T
```

Comment:
@bradzacher Nice shot! :tada:
IMO, that it would be better to move this file to `jsutils` directory.

Reply:
I thought about that, but I chose not to because the structure of the folders reflected that of the actual repo.
I can move if it you think it's best though.

Reply:
I think it would be confusing to have it inside `jsutils` since it doesn't match any file from `graphql-js`. If someone decides to compare TS typings to `graphql-js` file by file he would be puzzled by such file.
Plus when you open almost any `*.d.ts` you will see one extra import compared to the same file from `graphql-js`.
I would suggest making it named export and host under `types/graphql/tsutils.d.ts`  that way it would be clear that it something TS specific.
I also suggest making `import { Maybe } from "../tsutils";` the first line in a file separated by an empty line from rest of file.

Reply:
@IvanGoncharov Agree!

If someone wants to use `Maybe`, he will know that it's TS specific.

```ts
import { Maybe } from "graphql/tsutils";
```